### PR TITLE
fetch layer commands for local images

### PR DESCRIPTION
**Pull request by akpsgit**

### CODE REVIEW

File: docker/docker.go [hunk 1]
```diff
@@ -408,6 +431,20 @@ func newDockerImage(ctx context.Context, imageName string, option fanal_types.Do
 	return nil, func() {}, result
 }
 
+func newLocalDockerImage(imageName string) (containerregistry_v1.Image, error) {
+	ref, err := name.ParseReference(imageName, []name.Option{}...)
```
Comment (FrimIdan) [inline docker/docker.go]: what about
```
	if option.NonSSL {
		nameOpts = append(nameOpts, name.Insecure)
	}
```
options.NonSSL = config.InsecureRegistry

you are creating 
```
		opts := fanal_types.DockerOption{
			Timeout:               90 * time.Second,
			SkipPing:              true,
			UserName:              config.User,
			Password:              config.Password,
			NonSSL:                config.InsecureRegistry,
			InsecureSkipTLSVerify: config.InsecureTLS,
		}
```
for the `newRemoteDockerImage` - I think you can use it also here in `newLocalDockerImage`

Comment (akpsgit) [reply docker/docker.go]: Not sure I understand, these options are relevant only for the registry. Here it's local. Are you talking about []name.Option{}...?

Comment (FrimIdan) [reply docker/docker.go]: yes I know, from `/Users/idanf/gocode/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20201028122920-48a25d865182/image/image.go` it seems that `name.Insecure` is related also to local (or maybe I'm wrong)
```
func newDockerImage(ctx context.Context, imageName string, option types.DockerOption) (v1.Image, func(), error) {
	var result error

	var nameOpts []name.Option
	if option.NonSSL {
		nameOpts = append(nameOpts, name.Insecure)
	}
	ref, err := name.ParseReference(imageName, nameOpts...)
	if err != nil {
		return nil, func() {}, xerrors.Errorf("failed to parse the image name: %w", err)
	}

	// Try accessing Docker Daemon
	img, cleanup, err := daemon.Image(ref)
```

File: docker/docker.go [hunk 2]
```diff
@@ -294,34 +296,57 @@ func (i *Image) GetFsCommands() []*FsLayerCommand {
...
 	if err != nil {
-		return fmt.Errorf("failed to get docker image: %v", err)
+		return fmt.Errorf("failed to fetch layer commands: %v", err)
+	}
+	i.FsCommands = fsCommands
+
+	return nil
+}
+
+// FetchFsCommands retrieves information about image layers commands from docker registry.
```
Comment (Erezf-p) [inline docker/docker.go]: update comment

File: docker/docker.go (hunk 1 above)
Comment (akpsgit) [reply docker/docker.go]: Not sure what it does, but no problem

**Review by FrimIdan — APPROVED** [review]

**Pull request merged**